src/sentry/api/endpoints/project_plugins.py

```diff
@@ -13,6 +13,6 @@ def get(self, request, project):
         context = serialize([
             plugin
             for plugin in plugins.configurable_for_project(project, version=None)
-            if plugin.has_project_conf()
+            if plugin.has_project_conf() or plugin.get_plugin_type() == 'release-tracking'
```

Comment:
The problem is adding `has_project_conf` to freight will create a link to a configuration page that won't exist because it doesn't have a project conf page. So this is to make the necessary html be returned on the release-tracking page without adding that to freight confusingly/unnecessarily.